src/components/AudioPlayer.tsx

```diff
@@ -97,11 +99,15 @@ const AudioPlayer = ({ audio, disabled, isTtsText = false }: AudioPlayerProps):
 
   return (
     <VolumeIcon
-      disabled={disabled || !isInitialized}
+      disabled={disabled || !isInitialized || isSilent}
       isActive={isActive}
       onPress={handleSpeakerClick}
       testID='audio-player'>
-      <VolumeUpCircleIcon width={theme.spacingsPlain.lg} height={theme.spacingsPlain.lg} />
+      {isSilent ? (
+        <VolumeDisabled width={theme.spacingsPlain.md} height={theme.spacingsPlain.md} />
```

Comment:
Just a small recommendation. I think we started once with using React Native Paper which is similar to MUI
There are several nice and lean component such as a tooltip.
Not sure if that fits design needs

```
        <Tooltip enterTouchDelay={0} title={getLabels().general.error.deviceIsMuted} leaveTouchDelay={2600}>
        <VolumeDisabled width={theme.spacingsPlain.md} height={theme.spacingsPlain.md} />
        </Tooltip>
```